mlrun/api/db/base.py

```diff
@@ -218,53 +204,53 @@ def list_artifacts_v2(
         iter: int = None,
         best_iteration: bool = False,
         as_records: bool = False,
+        uid=None,
+        producer_id=None,
     ):
         pass
 
-    def read_artifact_v2(
-        self,
-        session,
-        key: str,
-        project: str = None,
-        iter: int = None,
-        producer_id: str = None,
-        tag: str = None,
-        uid: str = None,
-        raise_on_not_found: bool = True,
-    ):
-        pass
-
-    def del_artifact_v2(
+    @abstractmethod
+    def del_artifact(
         self, session, key, tag="", project="", uid=None, producer_id=None
     ):
         pass
 
-    def del_artifacts_v2(
-        self, session, name="", project="", tag=None, labels=None, ids=None, tree=None
+    @abstractmethod
+    def del_artifacts(
+        self,
+        session,
+        name="",
+        project="",
+        tag="*",
+        labels=None,
+        ids=None,
+        producer_id=None,
     ):
         pass
 
-    def list_artifact_tags_v2(
+    def list_artifact_tags(
         self, session, project, category: mlrun.common.schemas.ArtifactCategories = None
     ):
         pass
 
-    def overwrite_artifacts_with_tag_v2(
+    def store_artifact_v1(
         self,
         session,
-        project,
-        tag,
-        identifiers: typing.List[mlrun.common.schemas.ArtifactIdentifier],
+        key,
+        artifact,
+        uid,
+        iter=None,
+        tag="",
+        project="",
+        tag_artifact=True,
     ):
+        """
+        Store artifact v1 in the DB, this is the deprecated legacy artifact format
+        and is only left for testing purposes
+        """
         pass
 
-    def append_tag_to_artifacts_v2(
-        self,
-        session,
-        project: str,
-        tag: str,
-        identifiers: List[mlrun.common.schemas.ArtifactIdentifier],
-    ):
+    def read_artifact_v1(self, session, key, tag="", iter=None, project=""):
```

Comment:
Add comment and mark as deprecated (I assume this is also just kept here for testing)